Makefile

```diff
@@ -4,12 +4,13 @@ CFLAGS := -std=c99 -Wall -pedantic -Wextra -Werror ${CFLAGS}
 LDLIBS  = -ljansson
 INSTALL=install
 DESTDIR?=/
```

Comment:
While at it, it might be good to make DESTDIR default to empty.  Otherwise you wind up with paths beginning with `//` which at best is ugly, and at worst breaks on some platforms (Cygwin).